sqlx/tslint.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": ["../tslint.json"],
   "rules": {
+    "tsr-detect-non-literal-fs-filename": false,
     "tsr-detect-possible-timing-attacks": false,
-    "tsr-detect-non-literal-fs-filename": false
+    "tsr-detect-unsafe-regexp": false
```

Comment:
Just to confirm, this is OK because the SQLX stuff is run inside the VM sandbox?